Formula/gwsocket.rb

```diff
@@ -0,0 +1,22 @@
+class Gwsocket < Formula
+  desc "Simple, standalone, RFC6455 compliant WebSocket server"
+  homepage "http://gwsocket.io/"
+  url "https://github.com/allinurl/gwsocket/archive/v0.2.tar.gz"
+  sha256 "cafb0cbde1d8b366c74c1602a8d2e1bde054834aa5c618cfad5603a871ae4082"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+
+  def install
+    system "autoreconf", "-fvi"
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/gwsocket", "--version"
```

Comment:
I don't have a mac to test this out but I think a more substantial test could be something like this:

```
test do
    system "#{bin}/gwsocket", "--version"
    pid = Process.fork { exec "#{bin}/gwsocket", "--port=7890", "--echo-mode" }
    sleep 2

    begin
        assert_equal("HTTP/1.1 101 Switching Protocols\n", system("curl -i -N -H 'Connection: Upgrade' -H 'Upgrade: websocket' -H 'Host: localhost:7890' -H 'Sec-WebSocket-Key: YiINBlLGzYXLX3RfV7fUfw==' localhost:7890"))
    ensure
        Process.kill("SIGINT", pid)
        Process.wait(pid)
    end 
end
```